index.html

```diff
@@ -5,254 +5,299 @@
-        <h3 class="project-name">AI blog writer</h3>
-        <ul class="project-details">
-          <li class="lang">HTML/CSS</li>
-          <li class="lang">Ruby on Rails</li>
-          <li class="lang">JavaScript</li>
-        </ul>
-        <div class="button">
-          <button type="button" class="see-project">
-            See This project <img class="arrow" src="arrow.png" alt="arrow" />
+    <main>
+      <section class="headline" id="hello-section">
+        <div class="left">
+          <h1 id="author">
+            Shaheer <br />
+            Chaudry
+          </h1>
+          <p id="bio">
+            Hello! I am a software developer! I can help you build a product,
+            feature or website. Take a look at my work. If you like what you see
+            and have a project you need coded, don’t hesitate and contact me.
+          </p>
+          <button id="start-collaboration" type="button">
+            Start collaboration
           </button>
         </div>
-      </div>
+        <div class="right"></div>
+      </section>
 
-      <div class="project-info">
-        <img class="thumbnails" src="dell.png" alt="project 1" />
-        <h3 class="project-name">Wordpress Development</h3>
-        <ul class="project-details">
-          <li class="lang">HTML/CSS</li>
-          <li class="lang">Ruby on Rails</li>
-          <li class="lang">JavaScript</li>
-        </ul>
-        <div class="button">
-          <button type="button" class="see-project">
-            See This project <img class="arrow" src="arrow.png" alt="arrow" />
-          </button>
+      <section class="projects" id="portfolio-section">
+        <div id="projects-title">
+          <h1>Projects</h1>
         </div>
-      </div>
-
-      <div class="project-info">
-        <img class="thumbnails" src="something.png" alt="project 1" />
-        <h3 class="project-name">Curios guy</h3>
-        <ul class="project-details">
-          <li class="lang">HTML/CSS</li>
-          <li class="lang">Ruby on Rails</li>
-          <li class="lang">JavaScript</li>
-        </ul>
-        <div class="button">
-          <button type="button" class="see-project">
-            See This project <img class="arrow" src="arrow.png" alt="arrow" />
-          </button>
-        </div>
-      </div>
-      <div class="project-info">
-        <img class="thumbnails" src="macpro.png" alt="project 1" />
-        <h3 class="project-name">IOS simulator</h3>
-        <ul class="project-details">
-          <li class="lang">HTML/CSS</li>
-          <li class="lang">Ruby on Rails</li>
-          <li class="lang">JavaScript</li>
-        </ul>
-        <div class="button">
-          <button type="button" class="see-project">
-            See This project <img class="arrow" src="arrow.png" alt="arrow" />
-          </button>
+        <div class="one">
+          <div class="thumbnail"></div>
+          <div class="project-summary">
+            <h2>Project One</h2>
```

Comment:
- [ ] Kindly remove the white space from the right side of the image. The image should be aligned with the right side of the page with no margin on the right and also give a gap between the left and right sides of this section.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
 ![p2](https://user-images.githubusercontent.com/70483835/166121805-b653323b-94b1-4700-8042-980d16e7c9dc.png) | ![p1](https://user-images.githubusercontent.com/70483835/166121810-90dc69b7-0c6f-477d-8460-2b02403dbe51.png)